s-port_flashing.md

```diff
@@ -1,2 +1,108 @@
-# S-PORT flashing
-TO-DO
+# S.Port Firmware Flashing
+
+## File Preparation
+
+- Download the firmware
+
+  ![](images/sport-flash-001-download-firmware.png)
+  
+- Place the firmware on the SDCARD in the FIRMWARE folder
+  
+  ![](images/sport-flash-104-sd-hc-firmware.bmp)
+
+- Ensure that the file name is 33 characters or less including the dot and extension
+
+  ![](images/sport-flash-106-sd-hc-firmware-file-selected.bmp)
+  
+## Hardware Preparation
+
+### All transmitters except for Taranis Q X7 shipped with bottom S.Port pins
+
+- The external module bay’s +ve pin has the same voltage as the transmitter’s battery
+
+  Check that the device being flashed is rated for the transmitter’s battery voltage
+
+  If it is not then either change the battery or add a BEC in between the module bay’s  and device’s power pins
+
+- Use a servo cable with the positive and negative wires swapped at one end
+
+  ![](images/sport-flash-002-modified-servo-cable-small.jpg)
+
+- Plug the modified servo cable on to the external module bay pins with the signal wire on the pin in the corner
+
+  ![](images/sport-flash-003-ext-bay-small.jpg)
+
+- Plug the other end of the modified servo cable onto the S.Port pins of the device to be flashed
+
+  ![](images/sport-flash-004-gps-small.jpg)
+
+### Taranis Q X7 shipped with bottom S.Port pins
+
+- Use a regular servo cable
+
+- Plug one end into the bottom S.Port
+
+- Plug the other end onto the S.Port pins of the device to be flashed
+
+### Taranis Q X7 bottom S.Port pins added
+
+- Will not work
+
```

Comment:
Remove these two lines and add a something like

Note: Adding manually the Sport pins to a X7 variant shipped without the sport pins will not work as this requires also a newer PCB revision.

to the non sport section.